src/app/services/auth.service.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { JwtDto } from '../models/jwt-dto';
+import { LoginUser } from '../models/login-user';
+import { NewUser } from '../models/new-user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  auth = 'https://be-dev-yourney.herokuapp.com/auth/'
+  //auth = 'http://localhost:3000/'
+  constructor(private httpClient: HttpClient) { }
+
+  public new(newUser: NewUser): Observable<any>{
+    return this.httpClient.post<any>(this.auth + 'new', newUser);
+  }
+
+  public login(loginUser: LoginUser): Observable<any>{
+    return this.httpClient.post<JwtDto>(this.auth + 'login', loginUser);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
    return this.httpClient.post<JwtDto>(this.auth + "login", loginUser);
```